Source/Controllers/Settings.swift

```diff
@@ -14,10 +14,97 @@ class SettingsViewController: ViewController {
     @IBOutlet weak var usernameLabel: UILabel!
     @IBOutlet weak var tableView: UITableView!
 
+    private lazy var cells = [
+        [
+            Cell(
+                identifier: .regular,
+                action: logOut,
+                configure: {
+                    $0.textLabel?.text = "Logout"
+                    $0.textLabel?.textColor = .dangerTextColor
+                }
+            ),
+            Cell(
+                identifier: .regular,
+                action: deleteAccount,
+                configure: {
+                    $0.textLabel?.text = "Delete account"
+                    $0.textLabel?.textColor = .dangerTextColor
+                }
+            )
+        ],
+        [
+            Cell(
+                identifier: .detail,
+                action: nil,
+                configure: {
+                    $0.textLabel?.text = "Version"
+                    $0.textLabel?.textColor = .textColor
+                    $0.isUserInteractionEnabled = false
+
+                    let info = Bundle.main.infoDictionary
+                    let appVersion = info?["CFBundleShortVersionString"] as? String ?? "Unknown"
+                    let buildNumber = info?["CFBundleVersion"] as? String ?? "Unknown"
+                    $0.detailTextLabel?.text = "\(appVersion) (\(buildNumber))"
+                }
+            )
+        ]
+    ]
+
+    struct Cell {
```

Comment:
Lets move this struct to extension in same file at least. Otherwise controller becomes huge and unreadable. It also can be declared out of controller scope in separate file in Cells folder. It's up to you which variant is better.

Reply:
Done 👍